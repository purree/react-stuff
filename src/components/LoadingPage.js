import React from 'react';

const LoadingPage = () => (
    <div className="loader">
        <img className="loader__image" src="/pictures/loader.gif" alt="Loading spinner" />
    </div>
);

export default LoadingPage;
