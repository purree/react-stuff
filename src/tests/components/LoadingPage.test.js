import LoadingPage from '../../components/LoadingPage';
import React from 'react';
import { shallow } from 'enzyme';

test('should render the loading page', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
});