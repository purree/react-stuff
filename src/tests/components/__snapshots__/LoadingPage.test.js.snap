// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the loading page 1`] = `
<div
  className="loader"
>
  <img
    alt="Loading spinner"
    className="loader__image"
    src="/pictures/loader.gif"
  />
</div>
`;
